recipe/build_libsigcpp.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 set -ex
 
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./etc
```

Comment:
@jakirkham I think this should say `./build`, not `./etc`

```suggestion
cp $BUILD_PREFIX/share/gnuconfig/config.* ./build
```

Reply:
Ah ok. Copied from a different feedstock. So yeah may be different here. Thanks for catching that!

Sounds like you have already fixed this in a subsequent PR. So will leave this alone